src/ContentTemplates/Parameters/TicketParameters.php

```diff
@@ -148,6 +148,10 @@ protected function defineValues(CommonDBTM $ticket): array
         $items_ticket = Item_Ticket::getItemsAssociatedTo($ticket::getType(), $fields['id']);
         foreach ($items_ticket as $item_ticket) {
             $itemtype = $item_ticket->fields['itemtype'];
+            if (!class_exists($itemtype)) {
```

Comment:
I'm not totally convinced; since real issues (eg. typos) would be hidden that way

Reply:
If we must handle cases where a plugin is inactive or uninstalled, then I don't see any perfect solution. We may restrict this test to classes which belong to a plugin. But a typo may still be hidded in the plugin itself.

Reply:
There are many cases in GLPI where we silently ignore unknown class, but maybe the best way to do is to add a warning. It may generate support plugins that are not correctly cleaned, but silent failures are never a good idea, as it may hide  bugs and complicate debug.

```suggestion
            if (!class_exists($itemtype)) {
                trigger_error(sprintf('No class found for type %s', $itemtype), E_USER_WARNING);
```

Reply:
Ok to have a message in logs.